### ISSUE TITLE
Python install script is non standard, documentation needs clarification

### ISSUE THREAD

**Issue by datakid** [luntergroup/octopus#36]
**Describe the bug**

I'm about to install. I thought I'd use the python3 method.

Reading the instructions, looking for --prefix, find:
```
By default this installs to /bin relative to where you installed octopus. To install to a root directory (e.g. /usr/local/bin) use:

$ ./scripts/install.py --root
```

Not 100% sure what this means? Should I run 

```
$ ./scripts/install.py --root /path/to/desired/prefix
```
Or do I not actually get a choice and 

```
$ ./scripts/install.py --root
```
is my only option, and it requires that I install it into /usr/local/bin ?

If it's the former, would be great if it could be changed to the standard --prefix nomenclature.

Regardless of which, the documentation could be clearer. 

**Server (please complete the following information):**
 - OS: CentOS 
 - Version 7.5.5

**Comment by dancooke** [comment]
Thanks for the feedback. The latter is correct; it only installs into `/usr/local/bin`. I agree this is confusing and non-standard. I'll change it for the next release.

**Comment by dancooke** [comment]
I think that I've addressed this in 18b36eaaf789d294ddfe3514fbb0d4e03c4eeccd. These changes will be pushed into the next release and is available on develop branch. Please re-open if you run into any issues. Thanks again.
